src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/ExceptionsPipelineTests.cs

```diff
@@ -34,6 +34,43 @@ public sealed class ExceptionsPipelineTests
             "Microsoft.Diagnostics.Monitoring.StartupHook",
             TargetFrameworkMoniker.Net60);
 
+        private static ExceptionFilter simpleInvalidOperationException = new()
```

Comment:
Is simple vs full name still applicable?

Reply:
Strictly speaking, no...I kept variants of these tests to make sure that all the logic for not allowing partial names had been removed (basically just confirming that the functionality was correctly pulled out). They're still in there, but I can pull these out if you'd like since it's probably not something we have to repeatedly test for regressions.